simple_salesforce/api.py

```diff
@@ -140,6 +140,24 @@ def __init__(
                                  version=self.sf_version))
         self.apex_url = ('https://{instance}/services/apexrest/'
                          .format(instance=self.sf_instance))
+        self.tooling_url = ('{base}tooling/').format(base=self.base_url)
+
+    def execanon(self, apex_string):
+        """Executes a string of Apex code.
+        """
+        url = self.tooling_url + "executeAnonymous/"
+        params = {'anonymousBody': apex_string}
+        result = self.request.get(url, headers=self.headers, params=params)
```

Comment:
👍  to that; I have to override `_call_salesforce` in my codebase so that I can hook into some request-tracking code in my codebase. The consistency made it really easy